csatar-plugins/csatar/models/Scout.php

```diff
@@ -30,7 +30,7 @@ class Scout extends Model
         'personal_identification_number' => 'required',
         'gender' => 'required',
         'is_active' => 'required',
-//        'legal_relationship' => 'required', // temporary removed until empty select issue is fixed
```

Comment:
Az ilyen validaciok jelenleg mas modelleken is ki vannak kommentezve, pl. Association. Nem lehetne azokat is visszatenni?

Reply:
De, visszateszem.